pyvista/plotting/themes.py

```diff
@@ -2914,6 +2918,28 @@ def lighting_params(self, config: _LightingConfig):  # numpydoc ignore=GL08
             raise TypeError('Configuration type must be `_LightingConfig`.')
         self._lighting_params = config
 
+    @property
+    def logo_file(self) -> Optional[pathlib.Path]:  # numpydoc ignore=RT01
+        """Return or set the logo file.
```

Comment:
Can we state the default behavior is the PyVista logo.  Probably too complex to describe why it is a default of `None`.

Reply:
Add a note